.isort.cfg

```diff
@@ -6,9 +6,8 @@ include_trailing_comma = true
 known_tests = pytest,unittest,factory
 known_demo = demo
 sections = FUTURE,STDLIB,THIRDPARTY,TESTS,FIRSTPARTY,DEMO,LOCALFOLDER
-known_first_party = unicef_commonlib
+known_first_party = unicef_locations
 multi_line_output = 0
```

Comment:
To be consistent with the other repos, we probably want to change this to `multi_line_output = 3`